content/en/universal_service_monitoring/_index.md

```diff
@@ -63,6 +63,36 @@ If you have feedback about what platforms and protocols you'd like to see suppor
 Enable Universal Service Monitoring in your Agent by using one of the following methods depending on how your service is deployed and your Agent configured:
 
 {{< tabs >}}
+{{% tab "Operator" %}}
+
+To enable Universal Service Monitoring with the [Datadog Operator][1], update your `datadog-agent.yaml` manifest with the setting `spec.features.usm.enabled` set to `true` in the DatadogAgent resource:
+
+   ```yaml
+   apiVersion: datadoghq.com/v2alpha1
+   kind: DatadogAgent
+   metadata:
+     name: datadog
+   spec:
+     global:
+       credentials:
+        apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+        appSecret:
+         secretName: datadog-secret
+         keyName: app-key
+     features:
+       usm:
+         enabled: true
+   ```
+
+Note: Datadog Operator v1.0.0 or greater is required.
+
+<div class="alert alert-info">Support for GKE clusters running Google Container-Optimized OS (COS) is coming soon! In the Datadog Operator roadmap is introspection to automatically detect the cluster type allowing for seamless configuration and greater ease of use.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Support for GKE clusters running Google Container-Optimized OS (COS) is coming soon! An upcoming feature of Datadog Operator is introspection to automatically detect the cluster type, allowing for seamless configuration and greater ease of use.</div>
```
Is there a strong reason for including these upcoming features? We usually limit the public documentation to describing the current state of the product unless there is a compelling reason to do otherwise.

Reply:
Since the helm chart has a `providers.gke.cos` option and the datadog-operator doesn't, we wanted to indicate that support is not yet available, but coming soon. Should I change it to just say "Support for GKE clusters running Google Container-Optimized OS (COS) is not yet available."?